emails/2015/feb-meetup-call-for-speakers.md

```diff
@@ -0,0 +1,92 @@
+
+## How Django Boston works
+
+- Meetups happen on the 4th Thursday of the month (always at Akamai in Cambridge, unless otherwise noted)
+- Pizza and salad is always provided (by Akamai)
+- We often migrate to Mead Hall for drinks afterward (sponsored by another company)
+- Everyone is welcome: beginners, experts, intermediate, not-sure-how-to-categorize-yourself. EVERYONE!
+
+## Call for speakers
+
+This is the part of the email that's supposed to pump you up about giving a talk. To do so, I'll give you a quick story from my childhood:
+
+In kindergarten, my teacher asked me to give a talk on my favorite animal. I knew what it was (flying squirrel) so I was all like, "yeah, I got this." Well, as it happened, I got up to give the talk, and my mind went TOTALLY blank. It was the first time I had experienced this phenomenon, so I freaked out and ran back to my desk, but before I got there, I slipped on Lucy's Jello, flipped onto my back, and ripped out the seat of my pants. Sadly, nobody got to hear about the flying squirrel, and everybody laughed at me.
+
+The moral of the story: No matter what happens during your talk, it can't be the worst talk ever, cause I own that. And your talk will likely be THE BEST ONE EVER.
+
+But all jokes aside: we're all relying on the community as we make our living using freely available tools, and free knowledge generously shared by previous speakers. Please seriously consider helping the community by giving a talk (and making yourself look cool and seem smart, which you are!).
+
+Of course, you can give a talk on any topic you want (it should be about Django, or at least something interesting for Django programmers). If you don't know what you want to give a talk on, there are many ideas below. If you don't know what to do, but feel the nudge to share your knowledge, respond to the email, and one of us will help you figure it out. If you don't want to give a talk, but are interested in something, let us know.
+
+Alright, don't even think about closing the inbox, don't defer it, just respond and say "yes! I'll give a talk." :-)
+
+Ideas on finding a talk:
+
+1. Fork a talk
+
+Many, many talks have been given in the past, so instead of starting from the ground up, you could "fork" somebody else's talk and save a bunch of time, while still bringing lots of benefit to us. With a simple email, it's likely that you could get the permission of the original author to use their slides, code, etc.
+
+Places to find talks:
+
+Djangocon 2014: https://www.youtube.com/watch?v=cqP758k1BaQ&list=PLE7tQUdRKcybbNiuhLcc3h6WzmZGVBMr3
+Djangocon 2013: https://www.youtube.com/watch?v=drWhGUdadGw&list=PLtqtTJ4wP09YOFqm_lBCoQtmS6S0omW3J
+Djangocon Europe 2014: http://eventifier.com/event/djangoconeuropw/videos?page=2
+Search SpeakerDeck for Django: https://speakerdeck.com/search?utf8=%E2%9C%93&q=django (this brings up 44 pages of talks!)
```

Comment:
Another good place to find talks is http://pyvideo.org which essentially aggregates talks from every Python conference.